src/test/java/com/yandex/LoginPageTest.java

```diff
@@ -1,5 +1,60 @@
 package com.yandex;
 
+import org.junit.*;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.openqa.selenium.support.ui.WebDriverWait;
+import java.util.concurrent.TimeUnit;
+import static org.junit.Assert.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertAll;
+
 public class LoginPageTest {
+    public static WebDriver driver = null;
+    public static LoginPage loginPage;
+
+    @BeforeClass
+    public static void setup() {
+        System.setProperty("webdriver.chrome.driver", "C:\\Users\\Vadim\\Downloads\\chromedriver\\chromedriver.exe");
```

Comment:
Add chromedriver in PATH and remove this line, because if I (anybody) will try to execute your test, then I will need to change code.